projects/1.18/assets/andycats-flowers/andycats_flowers/lang/zh_cn.json

```
@@ -0,0 +1,141 @@
+{
+  "block.andycats_flowers.cyan_cosmos": "青色大波斯菊",
+  "block.andycats_flowers.purple_gladiolus": "紫色剑兰",
+  "block.andycats_flowers.tall_cyan_gladiolus": "高青色剑兰",
+  "block.andycats_flowers.pink_mallow": "粉红色锦葵",
+  "block.andycats_flowers.tall_white_daffodil": "高白色水仙",
+  "block.andycats_flowers.cyan_echinacea": "青色锥菊",
```

```suggestion
  "block.andycats_flowers.cyan_echinacea": "青色松果菊",
```
http://www.iplant.cn/info/%E6%9D%BE%E6%9E%9C%E8%8F%8A%E5%B1%9E